core/src/test/java/com/alibaba/druid/bvt/sql/oracle/create/OracleCreateViewTest13.java

```diff
@@ -55,7 +55,7 @@ public void test_types() throws Exception {
                         "WHERE rbi.rma_id > ?\n" +
                         "\tAND rsi.e_rprsendid = ?\n" +
                         "\tAND rsi.e_boardid IN (?)\n" +
-                        "FOR UPDATE(rbi.rma_id)",//
+                        "FOR UPDATE OF rbi.rma_id",//
```

Comment:
I don't find any place of Oracle doc saying it support `FOR UPDATE(col)` syntax, correct if I am wrong